conn/physic/units_test.go

```diff
@@ -536,3 +537,166 @@ func BenchmarkCelsiusFloatg(b *testing.B) {
 		buf.Reset()
 	}
 }
+
+func TestAtod(t *testing.T) {
+	const (
+		negative = true
+		postive  = false
+	)
+	tests := []struct {
+		s    string
```

Comment:
I'd prefer:
```
in       string
expected decimal
used     int
```

Then make one table for cases that succeeds, and one for a table that fails, preferably in two different test case. It'll be more readable.

Reply:
While rewriting the test noticed I was not returning the correct sign if it was minimum value. So now its more like:
```golang
if check < u || check > maxInt64 {
	if d.neg {
		return (-maxInt64), &parseError{err: errors.New("overflows minimum is"), s: "-" + maxUint64Str}
	}
	return maxInt64, &parseError{err: errors.New("overflows maximum is"), s: maxUint64Str}
}
```